pkg/libschc/patches/0001-fragmenter.c-drop-unused-variable.patch

```diff
@@ -0,0 +1,33 @@
+From 6ca9271090ed24fcd4e6e27a5751ade1020ad275 Mon Sep 17 00:00:00 2001
+From: Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+Date: Fri, 19 May 2023 23:34:12 +0200
+Subject: [PATCH] fragmenter.c: drop unused variable
```

Comment:
Want to create an upstream PR?

Reply:
https://github.com/imec-idlab/libschc/pull/44